tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -653,77 +793,29 @@ def _is_quote_char_dialect_mismatch(
         param("UNQUOTED_LOWER_COL", id="str UNQUOTED_LOWER_COL"),
+        param('"UNQUOTED_LOWER_COL"', id='str "UNQUOTED_LOWER_COL"'),
+        # DDL: UNQUOTED_UPPER_COL ----------------------------------
         param("unquoted_upper_col", id="str unquoted_upper_col"),
+        param('"unquoted_upper_col"', id='str "unquoted_upper_col"'),
         param("UNQUOTED_UPPER_COL", id="str UNQUOTED_UPPER_COL"),
+        param('"UNQUOTED_UPPER_COL"', id='str "UNQUOTED_UPPER_COL"'),
+        # DDL: "quoted_lower_col"-----------------------------------
         param("quoted_lower_col", id="str quoted_lower_col"),
+        param('"quoted_lower_col"', id='str "quoted_lower_col"'),
         param("QUOTED_LOWER_COL", id="str QUOTED_LOWER_COL"),
+        param('"QUOTED_LOWER_COL"', id='str "QUOTED_LOWER_COL"'),
+        # DDL: "QUOTED_UPPER_COL" ----------------------------------
         param("quoted_upper_col", id="str quoted_upper_col"),
+        param('"quoted_upper_col"', id='str "quoted_upper_col"'),
         param("QUOTED_UPPER_COL", id="str QUOTED_UPPER_COL"),
         param('"QUOTED_UPPER_COL"', id='str "QUOTED_UPPER_COL"'),
-        param('"quoted_lower_col"', id='str "quoted_lower_col"'),
-        param(
-            quoted_name(
-                "quoted_lower_col",
-                quote=None,
-            ),
-            id="quoted_name quoted_lower_col quote=None",
-        ),
-        param(
-            quoted_name(
-                "quoted_lower_col",
-                quote=True,
-            ),
-            id="quoted_name quoted_lower_col quote=True",
-        ),
-        param(
-            quoted_name(
-                "quoted_lower_col",
-                quote=False,
-            ),
-            id="quoted_name quoted_lower_col quote=False",
-        ),
-        param(
-            quoted_name(
-                "QUOTED_LOWER_COL",
-                quote=None,
-            ),
-            id="quoted_name QUOTED_LOWER_COL quote=None",
-        ),
-        param(
-            quoted_name(
-                "QUOTED_UPPER_COL",
-                quote=None,
-            ),
-            id="quoted_name QUOTED_UPPER_COL quote=None",
-        ),
-        param(
-            quoted_name(
-                "QUOTED_UPPER_COL",
-                quote=True,
-            ),
-            id="quoted_name QUOTED_UPPER_COL quote=True",
-        ),
-        param(
-            quoted_name(
-                "QUOTED_UPPER_COL",
-                quote=False,
-            ),
-            id="quoted_name QUOTED_UPPER_COL quote=False",
-        ),
-        param(
-            quoted_name(
-                "quoted_upper_col",
-                quote=None,
-            ),
-            id="quoted_name quoted_upper_col quote=None",
-        ),
     ],
 )
 class TestColumnIdentifiers:
-    # unskip to help debug `test_column_expectation` failures
-    @pytest.mark.skip(reason="sanity checks for helping debug issues")
     def test_raw_queries(
```

Comment:
I think we should get rid of this "test." It is not testing our functionality but rather the databases themselves. It might be nice to have a utility we can run against a database to give us these results if we're adding a new data source, but otherwise it can be confusing to the user about what the purpose of the test is. Also adds time and cost to these test runs, without a large amount of benefit as the underlying databases are unlikely to change their behavior.

Reply:
https://github.com/great-expectations/great_expectations/pull/8757#discussion_r1341393595